tests/FunctionalTestCase.php

```diff
@@ -282,17 +282,14 @@ protected function getPrimaryServer()
 
     protected function getServerVersion(ReadPreference $readPreference = null)
     {
-        $cursor = $this->manager->executeCommand(
+        $buildInfo = $this->manager->executeCommand(
             $this->getDatabaseName(),
             new Command(['buildInfo' => 1]),
             $readPreference ?: new ReadPreference(ReadPreference::RP_PRIMARY)
-        );
-
-        $cursor->setTypeMap(['root' => 'array', 'document' => 'array']);
-        $document = current($cursor->toArray());
+        )->toArray()[0];
 
-        if (isset($document['version']) && is_string($document['version'])) {
-            return preg_replace('#^(\d+\.\d+\.\d+).*$#', '\1', $document['version']);
+        if (isset($buildInfo->version) && is_string($buildInfo->version)) {
+            return preg_replace('#^(\d+\.\d+\.\d+).*$#', '\1', $buildInfo->version);
```

Comment:
Ah, it looks like the pattern here was subtly changed in https://github.com/mongodb/mongo-php-library/commit/2c80b35dbb0a4ccb2b1f0a678b81a3059271bfeb#diff-93b2fb11cb160b931a627169b6f5cb9c5d12b940e5a7ff1aec827f39abf1fc11R282 without being mentioned in the commit.

Reply:
Yep, I quietly fixed this there and created the DRIVERS ticket that spawned this PR. Of course I forgot to change the unified test runner 🤦‍♂️